manifests/pulp.pp

```diff
@@ -129,6 +142,39 @@
     }
   }
 
+  if $expose_mongodb {
+    include certs::mongodb
+    include certs::mongodb_client
+
+    Class['certs::mongodb_client'] -> Class['pulp']
+
+    $mongodb_bind_ip = $expose_mongodb_ipv6 ? {
+      true  => ['0.0.0.0', '::/0'],
+      false => ['0.0.0.0'],
+    }
+
+    class { 'mongodb::server':
+      bind_ip => $pulp::mongodb_bind_ip,
+      ipv6    => $pulp::mongodb_ipv6,
+      ssl     => true,
+      ssl_key => $certs::mongodb::mongodb_server_bundle,
+      ssl_ca  => $certs::mongodb::mongodb_server_ca_cert,
+      require => Class['certs::mongodb'],
+    }
+
+    $mongodb_seeds_real = "${facts['networking']['os']}:27017"
+    $mongodb_ssl_keyfile_real = $certs::mongodb_client::mongodb_client_key
+    $mongodb_ssl_certfile_real = $certs::mongodb_client::mongodb_client_cert
+    $mongodb_ca_path_real = $certs::mongodb_client::mongodb_client_ca_cert
+    $mongodb_ssl_real = true
+  } else {
+    $mongodb_seeds_real = 'localhost:27017'
```

Comment:
```suggestion
    $mongodb_seeds_real = $mongodb_seeds
```
This is breaking the existing test where `mongodb_seeds` is overridden at spec/classes/pulp_spec.rb:71 .